openhantek/src/glscope.cpp

```diff
@@ -212,10 +213,13 @@ void GlScope::initializeGL() {
           }
     )";
     const char *fshaderDesktop = R"(
-          #version 150
+          #version 120
           uniform highp vec4 colour;
-          out vec4 flatColor;
-          void main() { flatColor = colour; }
+          //out vec4 flatColor;
+          void main()
+          {
+          gl_FragColor = colour;
```

Comment:
See https://www.opengl.org/discussion_boards/showthread.php/184313-where-has-built-in-variable-gl_FragColor-gone-in-OpenGL-4-4?s=4da3cf47b4e9c44fa1c55d2e63528b85&p=1259897&viewfull=1#post1259897

gl_FragColor is deprecated and I rather not use it here to just support super old graphic stacks. Those old stacks can use the OpenGL ES 2.0 render path instead.